tests/operations/cargo.packages/remove_packages.json

```
@@ -0,0 +1,16 @@
+{
+    "args": [["git@1.1", "react", "test@1.1"]],
+    "kwargs": {
+        "present": false
+    },
+    "facts": {
+        "cargo.CargoPackages": {
+            "git": ["1.0"],
+            "react": [""],
+            "test": ["1.1"]
+        }
+    },
+    "commands": [
+        "cargo uninstall react test@1.1"
+    ]
```

```suggestion
    "args": [["git@1.1", "react", "test@1.1", "ansible"]],
    "kwargs": {
        "present": false
    },
    "facts": {
        "cargo.CargoPackages": {
            "git": ["1.0"],
            "react": [""],
            "test": ["1.1"],
            "ansible": ["1.2.1.0"]
        }
    },
    "commands": [
        "cargo uninstall react test@1.1 ansible"
    ]
```
Just adding the missing case where the installed package are a version but I assume that no version in the operation call means "any version".

Also, that Fact where `react` has no value in the list of versions, is it a valid case? I would expect that either a version is installed and then listed, or the package is not installed and hence not present at all in the Fact output...